src/components/Proposals/ProposalDetails/DisputableActionStatus.js

```diff
@@ -60,6 +65,34 @@ function DisputableActionStatus({ vote }) {
             )}
           </Item>
         )}
+        {scheduled && vote.settings.quietEndingPeriod && (
+          <Item heading="Quiet ending period">
+            <span>
+              Last {durationToHours(toMs(vote.settings.quietEndingPeriod))}{' '}
+            </span>
+            <span
+              css={`
+                color: ${theme.surfaceContentSecondary};
+              `}
+            >
+              hours
+            </span>
+          </Item>
+        )}
+        {scheduled && extendedPeriod && (
+          <Item heading="Quiet ending extension">
+            <span>
+              {durationToHours(toMs(vote.quietEndingExtensionDuration))}{' '}
+            </span>
+            <span
+              css={`
+                color: ${theme.surfaceContentSecondary};
+              `}
+            >
+              Hours
+            </span>
+          </Item>
+        )}
```

Comment:
Just checking, are these ordered correctly? The designs have them above the info box

![image](https://user-images.githubusercontent.com/11708259/92717156-adb13e80-f357-11ea-9b87-f6af7ac565ac.png)

Reply:
you are right! they should be after the timers